debian-setup.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+# Installs all of the necessary packages in a Docker container
+# MUST BE RUN WITH ROOT
+
+# Show all commands and exit upon failure
+set -eux
+
+# Make sure that all packages are up to date
+apt-get update
+apt-get upgrade -y
+
+# Install the official Debian packages that we need
+# curl and wget are not installed by default so this
+# is separate from the Clang/lld installation below
+# because we would need to at least install curl to
+# get LLVM's apt key
+apt-get install -y bc \
```

Comment:
would you mind putting bc on its own line? That way if we add something that begins with the letter `a`, we can insert it on its own line without having to update the `apt-get` line.

Reply:
Yes sure.